pontoon/administration/files.py

```diff
@@ -53,8 +66,8 @@ def split_comments_mine(
             cls.split_entities(
                 text, object, code=code, pointer=pointer, end=st0)
         groups = match.groups()
-        comment = silme.core.structure.Comment(
-            match.group(0)[1:].replace('\n#', '\n'))
+        comment = Comment()
+        comment.add(match.group(0)[1:].replace('\n#', '\n'))
```

Comment:
That's correct, I'm avoiding parsing comments, because it doesn't work as expected.

See:
https://bugzilla.mozilla.org/show_bug.cgi?id=1017849